wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WFileWidget.java

```diff
@@ -79,6 +88,14 @@ public void setFileTypes(final String[] types) {
 	public void setFileTypes(final List<String> types) {
 		getOrCreateComponentModel().fileTypes = types;
 	}
+	
+	/**
+	 * Checks if one or more file type is supplied.
+	 * @return True/False
+	 */
+	public boolean hasFileTypes() {
```

Comment:
This method name looks a lot better.

Reply:
This is not-picky but the Javadoc is incorrect. A boolean returns `true` or `false` not `True/False` and perhaps the returns snippet should indicate the conditions under which the return is either using the snippet `{@code true}` and/or `{@code false}`.